bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/i18n/ipcamera.properties

```diff
@@ -382,6 +382,8 @@ thing-type.config.ipcamera.hikvision.group.Image ch Settings.label = Image chann
 thing-type.config.ipcamera.hikvision.group.Image ch Settings.description = Settings for the image channel features which is not recommended to be used. See readme for more info.
 thing-type.config.ipcamera.hikvision.group.Settings.label = Main Settings
 thing-type.config.ipcamera.hikvision.group.Settings.description = Settings required to connect to the camera.
+thing-type.config.ipcamera.hikvision.hasExternalAlarmInputs.label=Has external alarm inputs.
+thing-type.config.ipcamera.hikvision.hasExternalAlarmInputs.description=Enable if the camera has external alarm inputs (such as PIR sensors etc.)
```

Comment:
Please synchronize label and description here with those in XML description.

Reply:
Simply run: mvn i18n:generate-default-tramslations

Reply:
Thanks for the hint: Done.